tests/testthat/test-wildcards.R

```diff
@@ -24,5 +24,35 @@ test_that("HTML `data-*` & `aria-* ` wildcards are passed along to layout approp
   expect_equal(x$props$`data-foo`, 1)
 })
 
+
+test_that("restricted pattern matching callback selectors must be formatted as a list", {
```

Comment:
It's OK to add these here, just wanted to note that the "wildcards" tested earlier in this script are the HTML wildcards, e.g. `data-*`, `html-*`.